cmd/fyne/commands/release.go

```diff
@@ -13,21 +11,131 @@ import (
 	"fyne.io/fyne/v2/cmd/fyne/internal/mobile"
 	"fyne.io/fyne/v2/cmd/fyne/internal/templates"
 	"fyne.io/fyne/v2/cmd/fyne/internal/util"
+	"github.com/urfave/cli/v2"
 )
 
-var (
-	macAppStoreCategories = []string{
-		"business", "developer-tools", "education", "entertainment", "finance", "games", "action-games",
-		"adventure-games", "arcade-games", "board-games", "card-games", "casino-games", "dice-games",
-		"educational-games", "family-games", "kids-games", "music-games", "puzzle-games", "racing-games",
-		"role-playing-games", "simulation-games", "sports-games", "strategy-games", "trivia-games", "word-games",
-		"graphics-design", "healthcare-fitness", "lifestyle", "medical", "music", "news", "photography",
-		"productivity", "reference", "social-networking", "sports", "travel", "utilities", "video", "weather",
-	}
-)
+var macAppStoreCategories = []string{
+	"business", "developer-tools", "education", "entertainment", "finance", "games", "action-games",
+	"adventure-games", "arcade-games", "board-games", "card-games", "casino-games", "dice-games",
+	"educational-games", "family-games", "kids-games", "music-games", "puzzle-games", "racing-games",
+	"role-playing-games", "simulation-games", "sports-games", "strategy-games", "trivia-games", "word-games",
+	"graphics-design", "healthcare-fitness", "lifestyle", "medical", "music", "news", "photography",
+	"productivity", "reference", "social-networking", "sports", "travel", "utilities", "video", "weather",
+}
 
-// Declare conformity to Command interface
-var _ Command = (*releaser)(nil)
+// Release returns the cli command for bundling release builds of fyne applications
+func Release() *cli.Command {
+	r := &releaser{}
+
+	return &cli.Command{
+		Name:  "release",
+		Usage: "Prepares an application for public distribution.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "target",
+				Aliases:     []string{"os"},
+				Usage:       "The operating system to target (android, android/arm, android/arm64, android/amd64, android/386, darwin, freebsd, ios, linux, netbsd, openbsd, windows)",
```

Comment:
What is `android` v.s `android/386` ?  Is that practically `android/arm386`?

Also, where did 32-bit BSD, Linux and Windows go?

Reply:
We probably shouldn't list all of possible values in the usage. The list we have is way too long.

Reply:
Makes sense. If extended helps comes up somewhere, we could link an FAQ or developer.fyne page that has information on building, or lists architectures.